src/super_gradients/training/utils/sg_trainer_utils.py

```diff
@@ -358,3 +359,18 @@ def get_callable_param_names(obj: callable) -> Tuple[str]:
     :return: Param names of that object
     """
     return tuple(inspect.signature(obj).parameters)
+
+
+def log_main_training_params(gpu_mode: MultiGPUMode, num_gpus: int, batch_size: int, batch_accumulate: int, len_train_set: int):
+    """Log training parameters"""
+    msg = "TRAINING PARAMETERS:\n" \
+          f"    - Mode:                         {gpu_mode.name if gpu_mode else None}\n"\
```

Comment:
please print "Single GPU" and not "None" in case a single GPU

Reply:
Chenged here e3045810dc583ab0e943275ca13f80614b17e785